docs/understanding-tracking-design/introduction-to-tracking-design/best-practices/index.md

```diff
@@ -0,0 +1,252 @@
+    "name": "view_product",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {},
+  "additionalProperties": false
+}
+```
+
+```json
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self": {
+    "vendor": "io.snowplow",
+    "name": "click_list_item",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {},
+  "additionalProperties": false
+}
+```
+:::
+
+</div>
+<div style={{ clear: "both" }} />
+
+
+### Recommendation 2: Avoid more granularity than necessary
+
+As stated in the previous recommendation, it is not desirable to group unrelated actions into a single event.
+However, when dealing with multiple events targeting the same type of action, it may be desirable to use a single event schema for them.
+Consider the following example.
+
+<div style={{ width: "50%", float: "left", paddingRight: "5px" }}>
+
+:::warning Bad
+Here we define two schemas covering the same type of action in different parts of the page.
+
+```json
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self": {
+    "vendor": "io.snowplow",
+    "name": "contact_button_click",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {},
+  "additionalProperties": false
+}
+```
+
+```json
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self": {
+    "vendor": "io.snowplow",
+    "name": "events_button_click",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {},
+  "additionalProperties": false
+}
+```
+:::
+
+</div>
+<div style={{ width: "50%", float: "left", paddingLeft: "5px" }}>
+
+:::tip Good
+Instead, it may be beneficial to use a single schema covering all button clicks on the page.
+This makes it easier to build reports that measure interaction across the whole page.
+It can also make it easier to capture additional button click events.
+
+```json
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "self": {
+    "vendor": "io.snowplow",
+    "name": "button_click",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "type": "object",
+  "properties": {
+    "name": {
+      "type": "string",
+      "description": "Name of the button"
+    }
+  },
+  "required": [ "name" ],
+  "additionalProperties": false
+}
+```
+:::
+
+</div>
+<div style={{ clear: "both" }} />
+
+The choice of the granularity of events should come from an analysis of the use case and reports that we want to produce.
+Some queries may benefit from having a higher granularity of the events.
+Nevertheless, keep in mind that grouping unrelated actions makes the event schemas more difficult to evolve independently and may introduce undesired dependencies.
```

Comment:
I think there is something missing at the end here about final steps e.g. record these in a tracking plan, go back to the original questions and make sure this data and structure can answer those questions, etc.

Reply:
Have added a section at the end to produce a tracking plan and also added a note to check that data answers the identified questions.